infrastructure/src/main/java/org/corfudb/infrastructure/LayoutServer.java

```diff
@@ -431,8 +435,9 @@ private void forceLayout(@Nonnull RequestMsg req, @Nonnull ChannelHandlerContext
     // TODO as this message is not set to ignore EPOCH.
     // TODO How do we handle holes in history if we let in layout commit message. Maybe we have a
     // TODO hole filling process
+    @VisibleForTesting
     @RequestHandler(type = RequestPayloadMsg.PayloadCase.COMMIT_LAYOUT_REQUEST)
-    public void handleCommitLayoutRequest(@Nonnull RequestMsg req, @Nonnull ChannelHandlerContext ctx,
+    void handleCommitLayoutRequest(@Nonnull RequestMsg req, @Nonnull ChannelHandlerContext ctx,
```

Comment:
These changes to the LayoutServer are addressing comments from the LayoutServer PR ?

Reply:
Yes they are